addons/l10n_vn/tests/test_l10n_vn_emv_qr.py

```diff
@@ -81,4 +81,20 @@ def test_emv_qr_vals(self):
         )
 
         # Check the whole qr code string
-        self.assertEqual(emv_qr_vals, '00020101021238590010A0000007270129000697042201156607040600001290208QRIBFTTA5204000053037045405100.05802VN5914company_1_data6007Vietnam62170113INV/TEST/0001630425A9')
+        self.assertEqual(emv_qr_vals, '00020101021238590010A0000007270129000697042201156607040600001290208QRIBFTTA52040000530370454031005802VN5914company_1_data6007Vietnam62170113INV/TEST/00016304E801')
+
+    def test_emv_qr_vals_with_accent_partner(self):
+        self.company_data['company'].partner_id.name = 'áÁéÉóÓíÍúÚýÝđĐ'
```

Comment:
I wonder if we didn't ignore many characters that may not work.

https://altcodeunicode.com/alt-codes-for-vietnamese-letters-with-accents-or-diacritics/